nevergrad/functions/base.py

```diff
@@ -254,6 +264,43 @@ def update_leaderboard(identifier: str, loss: float, array: np.ndarray, verbose:
         pass  # better avoir bugs for this
 
 
+def multi_experiments(
+    xps: tp.List[ExperimentFunction],
+    upper_bounds: tp.ArrayLike,
+    pareto_size: int,
+    no_crossval: tp.List[tp.Any] = [],
+) -> tp.List[ExperimentFunction]:
```

Comment:
I'll still need to understand this fully but here are a few things for now:
- instead of `xps` being xps and `no_crossval`  being indices, a more robust API would be to have the crossval xps on one side and the actual xps on another one.

Also, I would make this a `classmethod` of `MultiExperiment` with something like:

```python
@classmethod
def create_moo_crossvalidation_experiments(...)
```

Reply:
Done for the class method; genre of parametrization coming soon.

Reply:
Hum, I'm not sure for the parametrization... the solutions I see make the code significantly more complicated.